.github/workflows/ci.yml

```diff
@@ -37,6 +37,14 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run TypoScript lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+  codesniff:
+    name: "Codesniff"
+    runs-on: ubuntu-20.04
```

Comment:
This also should depend on the PHP linter so we don't try to sniff invalid PHP code.